src/Hazelcast.Net.Examples/Program.cs

```diff
@@ -136,12 +136,12 @@ private static void CollectUnobservedExceptions()
             GC.WaitForPendingFinalizers();
 
             // check for unobserved exceptions and report
-            var failed = false;
-            while (_unobservedExceptions.TryDequeue(out var args))
+            //var failed = false;
+            while (UnobservedExceptions.TryDequeue(out _))
```

Comment:
There is a comment above saying check exceptions and report. Should we do it here? As far as I see, we are just clearing the queue here, do not report anything back to the user

Reply:
we used to, and that is leftover code. removing.